libraries/ESP32/examples/RMT/RMTWriteNeoPixel/RMTWriteNeoPixel.ino

```diff
@@ -63,9 +63,15 @@ void loop()
         for (col=0; col<3; col++ ) {
             for (bit=0; bit<8; bit++){
                 if ( (color[col] & (1<<(8-bit))) && (led == led_index) ) {
-                    led_data[i].val = 0x80070006;
+                    led_data[i].level0 = 1;
+                    led_data[i].duration0 = 8;
+                    led_data[i].level1 = 0;
+                    led_data[i].duration1 = 1;
                 } else {
```

Comment:
These values are different,. a direct conversion between the `.val` and fields would be
```c++
led_data[i].level0 = 1;
led_data[i].duration0 = 7;
led_data[i].level1 = 0;
led_data[i].duration1 = 6;
```
Did you want to change the values?

Chuck

Reply:
Thank you chuck, good catch!

Indeed I meant to update the values (to be in line with the diagrams), but not that much. wanted to use pulses 8/4 instead of 7/6 (the led's are quite tolerant so it's accepted)